sw/device/lib/arch/device_fpga_nexysvideo.c

```diff
@@ -37,3 +39,11 @@ const uintptr_t kDeviceTestStatusAddress = 0;
 const uintptr_t kDeviceLogBypassUartAddress = 0;
 
 const bool kJitterEnabled = false;
+
+void device_fpga_version_print(void) {
```

Comment:
i like this idea!
although would `opentitantool` eventually need updates to understand that when talking to `silicon` it should not expect the bitstream printout?

Reply:
> i like this idea! 
Thanks!

> although would `opentitantool` eventually need updates to understand that when talking to `silicon` it should not expect the bitstream printout?

Yes, it has to distinguish between fpga and silicon, e.g. it will know it cannot load a bitstream or set the PLL of the silicon.